### DISCUSSION TITLE
How to send UTM data to analytics

### DISCUSSION THREAD

**Discussion by Rimantovas** [firebase/flutterfire#8456] [Q&A]
Hello, is it possible to send UTM values that we get from notifications and deep links that are sent through 3rd party services such as WebEngage, Facebook, etc. ? There seems to be a method called logCampaignDetails(), but only events sent from an android device are logged in analytics. When sending from an iOS device googleTagManager prints that it is processing the log event, but it never shows up in analytics.

**Comment by felixkwok2** [comment]
Did you find the solution?

I think I met a similar issue with you. Not sure is it the same case 

After calling "logCampaignDetails()", the events send from iOS device did not show up in analytics (in my case, it show up as Android)
https://github.com/firebase/flutterfire/issues/11958
